layouts/shortcodes/answer-key/keirbradfordgrey.html

```diff
@@ -0,0 +1,40 @@
+{{ $answer := .Get "answer" }}
```

Comment:
Are this and layouts/shortcodes/answer-key/eugenedepasquale.html still supposed to be here or are they leftover from a draft version of the code?

Reply:
These were leftovers and have been removed: https://github.com/spotlightpa/poor-richard/pull/604/commits/6f09b0d01086c57d6e976af86726b021cb75eb8e